docs/reference/modules/scripting.asciidoc

```diff
@@ -356,6 +356,8 @@ appropriate language.
 [float]
 === Groovy Sandboxing
 
+deprecated[1.6.0, the sandbox was previously turned off by default as it proved not effective]
```

Comment:
How about:

    "Groovy sandboxing has been disabled by default since 1.4.0(?) because it has proved to be ineffective. It will be removed completely in 2.0.0"